the-faseb-journal.csl

```diff
@@ -0,0 +1,149 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" initialize-with=", " names-delimiter=", " page-range-format="expanded" default-locale="en-US" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>The FASEB Journal</title>
+    <title-short>FASEB</title-short>
+    <summary>The Journal of the Federation of American Societies for Experimental Biology</summary>
+    <id>http://www.zotero.org/styles/the-faseb-journal</id>
+    <link href="http://www.zotero.org/styles/the-faseb-journal" rel="self"/>
+    <link href="http://www.zotero.org/styles/journal-of-analytical-toxicology" rel="template"/>
+    <link href="http://www.fasebj.org/site/misc/rc.xhtml" rel="documentation"/>
+    <author>
+      <name>Daniel Kraus</name>
+      <email>daniel.kraus@uni-wuerzburg.de</email>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="medicine"/>
+    <category field="biology"/>
+    <issn>0892-6638</issn>
+    <eissn>1530-6860</eissn>
+    <updated>2015-03-01T11:08:41+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="editor">
+    <names variable="editor">
+      <name name-as-sort-order="all" sort-separator=" " initialize-with="" delimiter=", " delimiter-precedes-last="always"/>
+      <label form="short" prefix=", "/>
+    </names>
+  </macro>
+  <macro name="author">
+    <group suffix=".">
+      <names variable="author">
+        <name and="text" delimiter-precedes-last="always" et-al-min="0" et-al-use-last="true" initialize-with=". " name-as-sort-order="all"/>
+        <label form="short" prefix=", "/>
+        <substitute>
+          <names variable="editor"/>
+          <text macro="title"/>
+        </substitute>
+      </names>
+    </group>
+  </macro>
+  <macro name="title">
+    <text variable="title" suffix="."/>
```

Comment:
you're already setting that period suffix below in the bibliography. Remove it here.